test.js

```diff
@@ -0,0 +1,79 @@
+const buttons=document.querySelectorAll('button');
+const rock=document.querySelectorAll('rock');
+const computerChoice=document.querySelector('.computerChoice');
+const userChoice=document.querySelector('.yourChoice');
+const winner=document.querySelector('.result');
+const score=document.querySelector('.score');
+const reset=document.getElementById("reset");
+let comScore=0;
+let userScore=0;

```

Comment:
✨P2: 앞서서 0:0 p 태그를 3개로 나눴다면, 조금 더 편리하게 자바스크립트 값에 접근하고 수정할 수 있습니다. 
```suggestion
const userScore = document.querySelector(".user-score");
const comScore = document.querySelector(".computer-score");
```